### DISCUSSION TITLE
Handlebar AST to original template string

### DISCUSSION THREAD

**Discussion by satyajitnayk** [handlebars-lang/handlebars.js#2017] [General]
Hey @jaylinski. I have implemented a method [here](https://github.com/satyajitnayk/handlebar-ast-to-str/blob/main/src/index.ts) converts handlebar AST back to original template string. 
I also have added several test cases to verify this & working on few more tests.

**Comment by jaylinski** [comment]
Nice work! Can you open a PR and add it to the readme (handlebars in the wild)?

**Reply by satyajitnayk** [reply]
sure! will do that @jaylinski

**Reply by satyajitnayk** [reply]
https://github.com/handlebars-lang/handlebars.js/pull/2019
